api/config/v1alpha1/envoygateway_types.go

```diff
@@ -80,23 +80,23 @@ type Provider struct {
 	// provides runtime configuration via the Kubernetes API.
 	//
 	// +optional
-	Kubernetes *KubernetesProvider `json:"kubernetes,omitempty"`
+	Kubernetes *GatewayKubernetesProvider `json:"kubernetes,omitempty"`
```

Comment:
shoudnt these be updated too with `EnvoyGateway` prefix ?

Reply:
did it @arkodg